bin/eslint.js

```diff
@@ -15,6 +19,37 @@ if (useStdIn) {
             exitCode = 1;
         }
     }));
+} else if (init) {
+    inquirer.prompt([
+        { type: "list", name: "indent", message: "What style of indentation do you use?", default: "tabs", choices: [{ name: "Tabs", value: "tabs" }, { name: "Spaces", value: 4}] },
+        { type: "list", name: "quotes", message: "What quotes do you use for strings?", default: "double", choices: [{ name: "Double", value: "double" }, { name: "Single", value: "single" }] },
+        { type: "list", name: "linebreak", message: "What line endings do you use?", default: "unix", choices: [{ name: "Unix", value: "unix" }, { name: "Windows", value: "windows" }]},
+        { type: "confirm", name: "semi", message: "Do you require semicolons?", default: true},
+        { type: "confirm", name: "es6", message: "Are you using ECMAScript 6 features?", default: false},
+        { type: "list", name: "env", message: "Where will your code run?", default: "browser", choices: [{ name: "Node", value: "node" }, { name: "Browser", value: "browser" }]},
```

Comment:
Ahh.. right, forgot that they are not mutually exclusive. Good thing inquirer has checkboxes prompt.